InnerEye/ML/Histopathology/datasets/panda_tiles_dataset.py

```diff
@@ -54,17 +54,26 @@ def __init__(self,
                  transform: Optional[Callable] = None,
                  **kwargs: Any) -> None:
         super().__init__(root=root, transform=transform)
-        self.base_dataset = PandaTilesDataset(root=root,
+        # filter out tiles based on occupancy threshold
+        self.full_dataset = PandaTilesDataset(root=root,
                                               dataset_csv=dataset_csv,
                                               dataset_df=dataset_df)
+        self.occupancy_threshold = 0
+        dataset_df_filtered = self.full_dataset.dataset_df.loc[self.full_dataset.dataset_df['occupancy'] > self.occupancy_threshold]  # type: ignore
+        dataset_df_filtered.reset_index(inplace=True)
+        self.base_dataset = PandaTilesDataset(root=root, dataset_df=dataset_df_filtered)
+        print(len(self.full_dataset))
+        print(len(self.base_dataset))
```

Comment:
As you said, it's better to remove these debugging `print`s.

Reply:
Done, thanks